pennylane/debugging/debugger.py

```diff
@@ -132,33 +132,198 @@ def pldb_device_manager(device):
 
 
 def breakpoint():
```

Comment:
Please can we have it so that `breakpoint` and `debug_x` all show up as top level in the documentation? Currently they are shown as within the `debugging` module (e.g., [here](https://xanaduai-pennylane--5789.com.readthedocs.build/en/5789/code/api/pennylane.debugging.breakpoint.html)), which is true but not the location we're recommending users access from.

Reply:
They appear both in the top level documentation and the module documentation. Let me try and remove the module documentation links.